cli/auth.go

```diff
@@ -660,7 +660,8 @@ func newCLIAuthFlowWithAuthDomain(authDomain, audience, clientID string, console
 	}
 }
 
-func (a *authFlow) loginAsUser(ctx context.Context) (*token, error) {
+func (a *authFlow) loginAsUser(c *cli.Context) (*token, error) {
+	ctx := c.Context
```

Comment:
For my own knowledge more than anything, why the change?

Reply:
Good question! The `cli.Context` has an `io.Writer` on it that allows us to write warning messages to terminal when connecting to the browser fails.